src/webgui/src/webgui/image_pub.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
This is a useful file, though not directly related to the webgui. Maybe we should put it in another package of utility nodes?